backend/app/settings.py

```diff
@@ -44,6 +44,7 @@
     'profiles',
     'corsheaders',
     'frontend',
+    'actstream',
```

Comment:
We do not need this anymore, we manually implemented the activity stream format without this library

Reply:
I removed this and the methods using actstream in app configs, thanks!